stripe.go

```diff
@@ -210,6 +213,38 @@ func SetBackend(backend SupportedBackend, b Backend) {
 
 // Call is the Backend.Call implementation for invoking Stripe APIs.
 func (s *BackendConfiguration) Call(method, path, key string, form *form.Values, params *Params, v interface{}) error {
+	return s.CallRaw(method, path, key, form, params, v)
+}
+
+// TODO: After every API call uses Call2, remove Call, then rename all
+// instances of Call2 to Call. This only exists as a separate method to keep
+// the build/tests working while we refactor.
+func (s *BackendConfiguration) Call2(method, path, key string, params ParamsContainer, v interface{}) error {
+	var body *form.Values
+	var commonParams *Params
+
+	if params != nil {
```

Comment:
we likely could/should enforce that params never be `nil` in the first place right?

Reply:
Yeah, we could go that direction too.

I was thinking though for calls like `*.Get` and even `CloseDispute` above, it might be kind of nice to be able to pass a `nil` because while you sometimes care about a parameter and want to pass a structure, you often don't. What do you think?

Reply:
That works for me!

Reply:
Ah yes I see how passing `nil` is easier/better in that case. I meant was that the method itself should not pass `nil` but ultimately it boils down to the same thing so agreed after all

Reply:
Excellent. I'll continue work on this after #578 is in!